scss/_modal.scss

```diff
@@ -15,6 +15,7 @@
   height: 100%;
   overflow-x: hidden;
   overflow-y: auto;
+  pointer-events: none;
```

Comment:
@patrickhlauke @ffoodd  I would like you opinion on this. :smiley:

Reply:
seems to all be working as intended

Reply:
Nope, it's not: the "Modal - Scrolling long content" demo from the documentation does not scroll anymore.

5.1.3: https://getbootstrap.com/docs/5.1/components/modal/#scrolling-long-content

vs this PR:

https://deploy-preview-35554--twbs-bootstrap.netlify.app/docs/5.1/components/modal/#scrolling-long-content

ping @GeoSot

Reply:
tried https://getbootstrap.com/docs/5.1/components/modal/#scrolling-long-content in Safari/iOS 15.4.1, Chrome 98/Android 11, Firefox 97/Android 11, as well as Chrome/Firefox/Edge latest on Windows. seem to all behave as they should @tkrotoff, unless I'm missing something? if there is a bug/problem here, suggest opening a new issue ticket

Reply:
ah, apologies @tkrotoff i did indeed read your comment backwards. i see what you mean now.

Reply:
Apologies, for my delayed answer. We are aware of this issue, and we are on discussion how to solve it, (or to revert this commit)

any suggestions, are welcome

Reply:
i suspect that for the first type of long modal (where the whole content of the overlay, which contains the actual modal itself, needs to scroll), there's no easy fix other than reverting the change here (as you effectively do want that container to be scrollable, so you can't neutralise `pointer-events` on it).

Reply:
I sadly tend to agree here... pity as it is a more clear approach

Reply:
I sadly tend to agree here. It is a pity for such a clear approach